docs/docs/cloud/quick_start.md

```diff
@@ -1,155 +1,194 @@
+          ```
+
+    === "Javascript"
 
-Learn more about the LangGraph CLI configuration file [here](./reference/cli.md#configuration-file).
+          ```js
+          {
+            "name": "my-app",
+            "packageManager": "yarn@1.22.22",
+            "dependencies": {
+              "@langchain/community": "^0.3.11",
+              "@langchain/core": "^0.3.16",
+              "@langchain/langgraph": "0.2.18",
+              "@langchain/anthropic": "^0.3.7"
+            }
+          }
+          ```
 
-5.  The `.env` file should have any environment variables needed to run your graph. This will only be used for local testing, so if you are not testing locally you can skip this step. NOTE: if you do add this, you should NOT check this into git. For this graph, we need two environment variables:
+1. __Create LangGraph configuration file__
+
+    The [`langgraph.json`][langgraph.json] file is a configuration file that describes what graph(s) you are going to deploy. In this case we only have one graph: the compiled `graph` object from `agent.py` / `agent.ts`.
+
+    === "Python"
+
+        ```json
+        {
+          "dependencies": ["."],
+          "graphs": {
+            "agent": "./agent.py:graph"
+          },
+          "env": ".env"
+        }
+        ```
+
+    === "Javascript"
+
+        ```json
+        {
+          "node_version": "20",
+          "dockerfile_lines": [],
+          "dependencies": ["."],
+          "graphs": {
+            "agent": "./src/agent.ts:graph"
+          },
+          "env": ".env"
+        }
+        ```
+
+    Learn more about the LangGraph CLI configuration file [here](./reference/cli.md#configuration-file).
+
+1. __Specify environment variables__
+
+    The `.env` file should have any environment variables needed to run your graph. This will only be used for local testing, so if you are not testing locally you can skip this step. 
+
+    !!! warning
+        The `.env` file should NOT be included with the rest of source code in your Github repository. When creating a deployment using LangGraph Platform, you will be able to specify the environment variables manually.
+
+    For this graph, we need two environment variables:
 
     ```shell
     ANTHROPIC_API_KEY=...
     TAVILY_API_KEY=...
     ```
 
-Now that we have set everything up on our local file system, we are ready to host our graph.
+!!! tip
+    Learn more about different application structure options [here](../how-tos/index.md#application-structure).
+
+Now that we have set everything up on our local file system, we are ready to test our graph locally.
+
+## Test the app locally
 
-## Test the graph build locally
+To test the LangGraph app before deploying it using LangGraph Platform, you can use [LangGraph Studio](../concepts/langgraph_studio.md) or start the [LangGraph server](../concepts/langgraph_server.md) locally.
 
-### Using LangGraph Studio Desktop (recommended)
+###  Using LangGraph Studio (Recommended)
```

Comment:
(1) I don't love that this is recommended given that it only supports mac os. I'd prefer removing this from the tutorial and doing something separate for langgraph studio
(2) if we're including it should only be LangGraph Studio Desktop, the fact that the web even works is going to confuse people

Reply:
okay i will 
* reorder to show CLI as the first option to verify
* keep only Studio desktop